man/resolved.conf.xml

```diff
@@ -225,6 +225,12 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>mDNS=</varname></term>
+        <listitem><para>Takes a boolean argument. If <literal>yes</literal> (the default), mDNS stack is enabled.
+        If <literal>no</literal>, the mDNS stack is disabled.</para></listitem>
```

Comment:
We need some explanation here how this meshes with the the per-network MulticastDNS= setting.

Also, in the commit message, please explain why this option only needs to add a config option, and why the `mdns_support` variable was already there, but how this changes existing logic. And why this option needed, when MulticastDNS= defaults to no in .network.